### ISSUE TITLE
Image Classifier Unit Test Failure

### ISSUE THREAD

**Issue by shantanuchhabra** [apple/turicreate#2744]
Image Classifier `text_export_coreml` unit test fails for model `VisionFeaturePrintSceneTest`.

```
=================================== FAILURES ===================================
________________ VisionFeaturePrintSceneTest.test_export_coreml ________________

self = <turicreate.test.test_image_classifier.VisionFeaturePrintSceneTest testMethod=test_export_coreml>

    def test_export_coreml(self):
        filename = tempfile.mkstemp('bingo.mlmodel')[1]
        self.model.export_coreml(filename)
    
        coreml_model = coremltools.models.MLModel(filename)
        self.assertDictEqual({
            'com.github.apple.turicreate.version': tc.__version__,
            'com.github.apple.os.platform': platform.platform(),
            'type': 'ImageClassifier',
            'version': '1'
            }, dict(coreml_model.user_defined_metadata)
        )
        expected_result = 'Image classifier (%s) created by Turi Create (version %s)' % (
                self.model.model, tc.__version__)
>       self.assertEquals(expected_result, coreml_model.short_description)
E       AssertionError: 'Image classifier (VisionFeaturePrint_Scene) created by Turi Create (version 6.0rc0)' != u'Image classifier (visionfeatureprint_scene) created by Turi Create (version 6.0rc0)'

test_image_classifier.py:185: AssertionError
```

**Comment by TobyRoseman** [comment]
@shantanuchhabra - Have we investigated this issue? Should this be fixed before the 6.0 release?

**Comment by shantanuchhabra** [comment]
No, this did not need to be fixed before 6.0. This is a unit test failure that has existed since before 5.8.
